docs/cbc_sdk.audit_remediation.rst

```diff
@@ -1,4 +1,4 @@
-Audit and Remediation Package
+cbc\_sdk.audit\_remediation package
```

Comment:
Why are these being reverted?

Reply:
All those files got regenerated when I rebuilt the docs.

Reply:
Please undo this change as we did this intentionally as well as any other package name title changes

Reply:
Fixed, but I wish the doc building process didn't step on those titles.